app/src/main/java/com/android/calendar/chips/CircularImageView.java

```diff
@@ -58,14 +59,17 @@ public CircularImageView(Context context, AttributeSet attrs, int defStyle) {
     @Override
     protected void onDraw(Canvas canvas) {
         Drawable drawable = getDrawable();
-        BitmapDrawable bitmapDrawable = null;
+        BitmapDrawable bitmapDrawable;
         // support state list drawable by getting the current state
         if (drawable instanceof StateListDrawable) {
-            if (((StateListDrawable) drawable).getCurrent() != null) {
-                bitmapDrawable = (BitmapDrawable) drawable.getCurrent();
-            }
+            drawable.getCurrent();
```

Comment:
why is this called here? get implies there is a return value but it is not used.

I don't understand this change.